### DISCUSSION TITLE
Request for helping loading dxf into Hypermesh

### DISCUSSION THREAD

**Discussion by MHLee808** [mozman/ezdxf#675] [Q&A]
Hello all!
Currently I'm have a difficulty for exporting a geometry via dxf format in ezdxf.
I'm trying to import exported dxf into Hypermesh to make FE mesh however I don't know why dxf file isn't imported to Hypermesh.
So, I just have tried to importing generated dxf file into UG-NX , it was successfully imported to UG.

Simply speaking, my dxf file that is created by ez dxf is working ok @ UG but that is not working @ Hypermesh.
Is there anyone who knows problem for this? I'm really appreciate if anyone give me a comment or hints on it.

I attached code as below
#-------------------------------------------------------------------------------------------------#
import ezdxf

doc = ezdxf.new("R2018")
msp = doc.modelspace()

points = section_pair[1]
msp.add_lwpolyline(points)

doc.saveas("lwpolyline10000.dxf")

section_pair[1] =  [(107, 167),
 (108, 167),
 (109, 167),
 (110, 167),
 (111, 167),
...
 (341, 167),
 (342, 167),
 (343, 167)]

**Comment by mozman** [comment]
Hi!

1. Try to get help from `Hypermesh` because it is obviously a problem of that application.
2. If you can't get help from `Hypermesh` post the error message of the application. If the application doesn't give you any error messages, I'm afraid I can't help you.
